docs/resources/anycast_host.md

```diff
@@ -0,0 +1,176 @@
+  config_bgp = {
+    asn           = "6500"
+    asn_text      = "6500"
+    holddown_secs = 180
+    neighbors = [
+      {
+        asn        = "6501"
+        ip_address = "10.20.0.3"
+      }
+    ]
+  }
+
+  # Adding the BGP configuration
+  config_ospf = {
+    area_type           = "STANDARD"
+    area                = "10.0.0.1"
+    authentication_type = "Clear"
+    interface           = "eth0"
+    authentication_key  = "YXV0aGV"
+    hello_interval      = 10
+    dead_interval       = 40
+    retransmit_interval = 5
+    transmit_delay      = 1
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `anycast_config_refs` (Attributes List) (see [below for nested schema](#nestedatt--anycast_config_refs))
+- `config_bgp` (Attributes) (see [below for nested schema](#nestedatt--config_bgp))
+- `config_ospf` (Attributes) (see [below for nested schema](#nestedatt--config_ospf))
+- `config_ospfv3` (Attributes) (see [below for nested schema](#nestedatt--config_ospfv3))
+
+### Read-Only
+
+- `created_at` (String)
+- `id` (Number) The ID of this resource.
+- `ip_address` (String) IPv4 address of the on-prem host
+- `ipv6_address` (String) IPv6 address of the on-prem host
+- `name` (String)
+- `updated_at` (String)
+
+<a id="nestedatt--anycast_config_refs"></a>
+### Nested Schema for `anycast_config_refs`
+
+Required:
+
+- `anycast_config_name` (String)
+
+Optional:
+
+- `routing_protocols` (List of String) Routing protocols enabled for this anycast configuration, on a particular host. Valid protocol names are "BGP", "OSPF"/"OSPFv2", "OSPFv3".
+
+
+<a id="nestedatt--config_bgp"></a>
+### Nested Schema for `config_bgp`
+
+Required:
+
+- `asn` (Number)
+- `holddown_secs` (Number)
+
+Optional:
+
+- `fields` (Attributes) (see [below for nested schema](#nestedatt--config_bgp--fields))
+- `keep_alive_secs` (Number)
+- `link_detect` (Boolean)
+- `neighbors` (Attributes List) (see [below for nested schema](#nestedatt--config_bgp--neighbors))
+- `preamble` (String) Any predefined BGP configuration, with embedded new lines; the preamble will be prepended to the generated BGP configuration.
+
+Read-Only:
+
+- `asn_text` (String) Examples:     ASDOT        ASPLAIN     INTEGER     VALID/INVALID     0.1          1           1           Valid     1            1           1           Valid     65535        65535       65535       Valid     0.65535      65535       65535       Valid     1.0          65536       65536       Valid     1.1          65537       65537       Valid     1.65535      131071      131071      Valid     65535.0      4294901760  4294901760  Valid     65535.1      4294901761  4294901761  Valid     65535.65535  4294967295  4294967295  Valid      0.65536                              Invalid     65535.655536                         Invalid     65536.0                              Invalid     65536.65535                          Invalid                  4294967296              Invalid
```

Comment:
description markdown not properly formatted

Reply:
done